package.json

```diff
@@ -70,6 +71,7 @@
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-standard": "^4.0.1",
     "husky": "^8.0.3",
+    "lerna": "^7.1.5",
```

Comment:
Inclusion of lerna is dubious at this point. We would have to do significant refactoring to make the repo more of a modular monorepo style with multiple packages being built & produced. I can remove this before merging.